src/Core/src/Handlers/Element/ElementHandler.cs

```diff
@@ -32,6 +32,8 @@ protected ElementHandler(IPropertyMapper mapper, CommandMapper? commandMapper =
 
 		public IElement? VirtualView { get; private protected set; }
 
+		internal bool _isPlatformViewNew;
```

Comment:
Maybe this can be an internal property:

```suggestion
		internal bool IsPlatformViewInitialized { get; }
```

But in a perfect world, where would this property go? Should this be on an interface or something to avoid the cast in each mapper? Pity we never added a `isFirstRun` property to the mappers... pity.

But maybe we just need a handler `IsInitialized` property to IViewHandler? Can we add that as a default interface implementation?